Content.Server/Destructible/Thresholds/Behaviors/SpawnEntitiesBehavior.cs

```diff
@@ -25,6 +27,9 @@ public sealed partial class SpawnEntitiesBehavior : IThresholdBehavior
         [DataField("transferForensics")]
         public bool DoTransferForensics = false;
 
+        [DataField("transferStickSurface")]
+        public bool DoTransferStickSurface = false;
+
```

Comment:
I'm kinda hesitant if we keep adding more datafields here and don't raise an event instead or the likes as this will quickly get bloated.

Reply:
Yeah, this could be issue. But it probably should be fixed in another PR

Reply:
I thought it would be out of scope. Okay, I'll do this in this PR

Reply:
done